templates/taxi/logout.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Logged Out</title>
+</head>
+<body>
+  <h1>You have been logged out</h1>
+  <p><a href="{% url 'login' %}">Log in again</a> or <a href="{% url 'home' %}">return to home page</a>.</p>

```

Comment:
The URL tag `{% url 'home' %}` assumes there is a URL named 'home'. Ensure that this URL name is defined in your `urls.py`. If the home page is the index view, it should match the name 'index'.